sli/slistartup.cc

```diff
@@ -188,14 +184,21 @@ SLIStartup::SLIStartup( int argc, char** argv )
   , exitcode_unknownerror_name( "unknownerror" )
   , environment_name( "environment" )
 {
-  ArrayDatum args_array;
+	// Initialize here instead of in initializer list to defer string concatenation
+	// until runtime to avoid incorrect path replacement by Conda. See #2237.
+	sliprefix = NEST_INSTALL_PREFIX;
+	slilibdir = sliprefix + "/" + NEST_INSTALL_DATADIR;
+	slidocdir = sliprefix + "/" + NEST_INSTALL_DOCDIR;
+	startupfile = slilibdir + "/sli/sli-init.sli";
```

Comment:
Replace tabs by spaces
```suggestion
  // Initialize here instead of in initializer list to defer string concatenation
  // until runtime to avoid incorrect path replacement by Conda. See #2237.
  sliprefix = NEST_INSTALL_PREFIX;
  slilibdir = sliprefix + "/" + NEST_INSTALL_DATADIR;
  slidocdir = sliprefix + "/" + NEST_INSTALL_DOCDIR;
  startupfile = slilibdir + "/sli/sli-init.sli";
```